src/main/resources/archetype-resources/pom.xml

```diff
@@ -11,6 +11,10 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!-- jms flags with IDE prefix -->
+        <addopenflags>${addopensflagsList}</addopenflags>
+        <!-- jms flags standard -->
+        <addopenfflags>${addopensfflagsList}</addopenflags>
```

Comment:
is there no way to avoid this somehow? Why does the maven archetype need to know about the "-J" list too?

Reply:
I will remove this, the nbm-application that may need the -J is for the other archetype.